examples/node_failure.yaml

```diff
@@ -9,6 +9,7 @@ metadata:
   name: node-failure
   namespace: chaos-demo
 spec:
+  level: node # note that any labels need to be applied on the nodes for this Disruption to work
```

Comment:
Actually, this change (and the one in the other node_failure example) are not correct. Your comment alludes to it, but `level` here really only affects the targeting of `selector`. `level: pod` is totally valid for a `node-failure` disruption kind. If you pull these two changes, I'll merge the rest of the doc fixes (which are really great, thank you!)

Reply:
Hm, maybe I misunderstood how this works. I also read [this](https://github.com/DataDog/chaos-controller/issues/223) issue.

My assumption was the following:
- When not specifying the level it defaults to pod
- If I add a label to the pod the expectation is it will disrupt the node where the pod is located

Unless I did something wrong this didn't work. I only managed to disrupt nodes after setting the level and labelling them.

Reply:
Both of those assumptions are correct. It's interesting that it didn't work. Do you have any logs from the controller manager at that time?

Reply:
Right, I tested that again and I missed something.
I deploy the controller in the `chaos-engineering` namespace and my applications in a different namespace.
I was setting the namespace to the one where the controller gets deployed but apparently it needs to match the target pod's one.

I've reverted the change.

Reply:
Just to add some context, the node failure is the only injector which does not use anything related to a container so the level only impacts the selector as @ptnapoleon said: https://github.com/DataDog/chaos-controller/blob/main/injector/node_failure.go

Reply:
Makes sense @Devatoria 👍